### PR TITLE
Update docs about build process

### PR DESCRIPTION
**Pull request by stsewd**
Describe each docker image used to build the docs

ref to https://github.com/rtfd/readthedocs.org/pull/4419#discussion_r204810381

### CODE REVIEW

File: docs/builds.rst [hunk 1]
```diff
@@ -53,17 +54,23 @@ Read the Docs cannot create a PDF version of your documentation with the *Mkdocs
...
 Then we build the proper backend code for the type of documentation you've selected.
 
-If you have the *Install Project* option enabled, we will run ``setup.py install`` on your package, installing it into a virtual environment. You can also define additional packages to install with the *Requirements File* option.
+If you have the *Install Project* option enabled, we will run ``setup.py install`` on your package, installing it into a virtual environment.
+You can also define additional packages to install with the *Requirements File* option.
 
-When we build your documentation, we run `sphinx-build -b html . _build/html`, where `html` would be replaced with the correct backend. We also create man pages and pdf's automatically based on your project.
+When we build your documentation, we run `sphinx-build -b html . _build/html`,
+where `html` would be replaced with the correct backend.
+We also create pdf's and ePub's automatically based on your project.
```
Comment (stsewd) [inline docs/builds.rst]: The rest of the lines are the same, they were just too long. Except here, here we have had `man pages`, but we don't build man pages anymore.

File: docs/builds.rst [hunk 2]
```diff
@@ -88,18 +95,66 @@ The artifacts should end up in ``self.version.project.artifact_path(version=self
...
+  * ``pip``
+:Other packages:
+  * ``libxml2-dev``
+  * ``libxslt1.1``
+  * ``libevent``
+  * ``textlive-full``
+  * ``graphviz``
+  * ``pandoc``
+  * ``doxygen``
+  * ``libjpeg``
```
Comment (stsewd) [inline docs/builds.rst]: Not sure if all these packages are relevant, I keep the most that were in the previous description

File: docs/builds.rst (hunk 1 above)
Comment (humitos) [reply docs/builds.rst]: Don't we need double ` for the command and `html` here?

Comment (stsewd) [reply docs/builds.rst]: Yeah

File: docs/builds.rst [hunk 3]
```diff
@@ -88,18 +95,66 @@ The artifacts should end up in ``self.version.project.artifact_path(version=self
...
-    * libxslt1.1
-    * libxml2-dev
+The build environment
+---------------------
+
+The build process is executed inside Docker containers,
+by default the image used is ``readthedocs/build:2.0``,
+but you can change that using a :doc:`configuration file <yaml-config>`.
+
+Each image is described below.
```
[the remark this answers is not in the document]
Comment (stsewd) [reply docs/builds.rst]: Good idea

File: docs/builds.rst [hunk 4]
```diff
@@ -102,7 +102,17 @@ The build process is executed inside Docker containers,
 by default the image used is ``readthedocs/build:2.0``,
 but you can change that using a :doc:`configuration file <yaml-config>`.
 
-Each image is described below.
+.. note::
+   
+   The Docker image does have a select number of C libraries installed,
+   because they are used across a wide array of python projects.
+   We can't install every C library out there,
+   but we try and support the major ones
```
Comment (humitos) [inline docs/builds.rst]: Final dot: `.`

Comment (stsewd) [reply docs/builds.rst]: :sob:

**Pull request merged**